scaffolding-chef/lib/scaffolding.sh

```diff
@@ -52,8 +52,8 @@ export SSL_CERT_FILE="{{pkgPathFor "core/cacerts"}}/ssl/cert.pem"
 cd {{pkg.path}}
 
 exec 2>&1
-exec chef-client -z -l {{cfg.log_level}} -c $pkg_svc_config_path/client-config.rb --once
-exec chef-client -z -i {{cfg.interval}} -s {{cfg.splay}} -l {{cfg.log_level}} -c $pkg_svc_config_path/client-config.rb
+chef-client -z -l {{cfg.log_level}} -c $pkg_svc_config_path/client-config.rb --once
+chef-client -z -i {{cfg.interval}} -s {{cfg.splay}} -l {{cfg.log_level}} -c $pkg_svc_config_path/client-config.rb
```

Comment:
I think you want to leave the 2nd exec since you'd want the supervisor to restart chef-client if the daemon dies.

Reply:
Just tested this, after killing that `chef-client` process the Supervisor will restart it as the run hook will exit. So I don't think the second exec is needed.

Reply:
You absolutely must exec. If you don't, you'll ahve multiple chef processes running.

Reply:
You must exec the last command correct? Removing the exec from the previous line should be okay?

Reply:
It is, because the default shell behavior is to ignore the error code. The process you want managed is the process that should be `exec` in the end.

Reply:
:+1: